src/onnx/onnx_parser.cpp

```diff
@@ -514,6 +514,26 @@ shape::type_t get_type(int dtype)
     }
 }
 
+bool is_type_float(int dtype)
```

Comment:
Why we require `int` overload ?

Reply:
artifact of something I had before. Ill remove it